google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -659,6 +678,18 @@ private void appendLoop() {
     log.info("Append thread is done. Stream: " + streamName + " id: " + writerId);
   }
 
+  private void throwIfWaitCallbackTooLong() {
+    Duration milliSinceLastCallback =
+        Duration.between(lastRequestCallbackTriggerTime, Instant.now());
+    if (milliSinceLastCallback.compareTo(MAXIMUM_REQUEST_CALLBACK_WAIT_TIME) > 0) {
+      throw new IllegalStateException(
```

Comment:
I don't think this is really an IllegalStateException

Reply:
Let's use a generic runtime exception, the goal is only to trigger uncaught exception handler